builtin/logical/postgresql/secret_creds.go

```diff
@@ -138,6 +151,10 @@ func (b *backend) secretCredsRevoke(
 		pq.QuoteIdentifier(username)))
 
 	revocationStmts = append(revocationStmts, fmt.Sprintf(
```

Comment:
Same change here re `fmt.Sprintf()` being prohibited, except it looks like this will have to be changed to pass around a set of parallel arrays, one for the queries and then a second for each query's args.